test/pkg/environment/common/setup.go

```diff
@@ -110,17 +109,6 @@ func (env *Environment) Cleanup(opts ...Option) {
 	env.ExpectNoCrashes()
 }
 
-func (env *Environment) ForceCleanup(opts ...Option) {
```

Comment:
Wait is there a reason we're not force cleaning anymore? I realize I asked to include Machine in Cleanable Objects, but I meant that we should try graceful cleanup, and if that doesn't work, then do Force cleanup.

Reply:
ForceCleanup was largely because we weren't confident that Provisioner propagation would catch everything due to races. Now that that isn't there anymore, I think it's fine to remove it and we are safe without force cleanup.

If that ends up not being the case, we can always add it back in later.